More info on why Python is good for ML. (#23623)
freeCodeCamp/freeCodeCamp: guide/english/python/what-is-python-used-for/index.md

##### Data Science / Analysis:

Python is the leading language of choice for many data scientists. Python has grown in popularity within the field due to the availability of many excellent libraries
focused on data analysis (for example NumPy and Pandas) and data visualisation (for example Matplotlib and Seaborn). Python really made Data crunching fun thanks to the wide selection of libraries available. Ipython with JupyterLab is another form of Python that improves the usage of Python in the field of data science.

##### Automation:

> Python is a very flexible language that can be used to automate boring or repetitive tasks. System administrators often use it by writing scripts which can be easily executed from the terminal. Python can also be used to create bots which automate some of our daily tasks.

Reviewer: ```suggestion
Python is a very flexible language that can be used to automate boring or repetitive tasks. System administrators often use it to write scripts which can be easily executed from the terminal. Python can also be used to create bots that automate some of our daily tasks.
```

##### Artificial Intelligence:
Python is also used extensively in the growing field of Artificial Intelligence (AI). Google selected Python to be one of the first well-supported programming languages for training and interacting with models using Tensorflow.

#### Mobile app development:
Mobile apps and games can be created with python using Kivy, Pygame and PyQt.

#### Security And Networking:

> Python is used to build Networking tools and security-tools that are widely used.Python's remote automation is the most secured, fast and efficient for cloud-testing of frameworks, that's why professional developers use python for making most secured frameworks and for socket-programmings.

Reviewer: ```suggestion
Python is used to build Networking and Security tools that are widely used. Python's remote automation is secured, fast and efficient for cloud-testing of frameworks, which is why professional developers use python for making secure frameworks, and for socket programming.
```

##### Machine Learning, Deep Learning:
Python is one of the best languages suited for machine learning, deep learning and data analytics with a stronghold in all of them.

> There are specialized languages best suited for various role such as R and MATLAB but when it comes to overlapping fields of applications python wins hands down due to its flexibility and rapid prototyping and availability of libraries. It is also has consistent syntax and is similar to the human language.

Reviewer: ```suggestion
There are specialized languages best suited for various roles, such as R and MATLAB, but when it comes to overlapping fields of applications, Python wins hands down due to its flexibility, rapid prototyping, and availability of libraries. It also has consistent syntax and is similar to the human language.
```
